iis-wf/iis-wf-export-actionmanager/src/test/java/eu/dnetlib/iis/wf/export/actionmanager/sequencefile/FieldAccessorTest.java

```diff
@@ -0,0 +1,145 @@
+package eu.dnetlib.iis.wf.export.actionmanager.sequencefile;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.fail;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.apache.cxf.helpers.IOUtils;
```

Comment:
Do we want such a dependency here? Apache Commons IOUtils could be more convenient.